### ISSUE TITLE
How i can open the App in All-Uread view?

### ISSUE THREAD

**Issue by hal0003** [Ranchero-Software/NetNewsWire#3487]
How can I set (iOS) that when I open the app, the app always opens in the All-Unread view?

The app always opens in the feeds-overview for me.

It used to work in the past. Suddenly no longer.

**Comment by vincode-io** [comment]
As long as you leave it on the All Unread feed it will reopen to it. 

You aren't force quitting NNW are you? That would cause it to forget what feed you were last viewing.

**Comment by hal0003** [comment]
Yes, I always close all apps completely via Taskmanager.

**Comment by vincode-io** [comment]
That's why it isn't opening in All Unread. When you force quit, you are telling iOS that you encountered a problem and had to force quit it. iOS will purge all state information in an effort to get the application to be responsive next time you launch it.

Force quitting apps also wastes battery and degrades system performance, since the application has to restart from scratch when it happens. Background sync is also disabled when when you force quit an app and won't work.

Basically you are doing extra steps on your phone to provide yourself with a worse experience when you force quit apps.
